### ISSUE TITLE
Update required

### ISSUE THREAD

**Issue by CAD97** [CAD97/SpawnerCraft2#7]
spawnedID in EntityEggInfo has been deprecated.
spawnCreature(World,int,double,double,double) in EntityEggInfo has been deprecated.

Use new system:

```
Build 1.8-11.14.3.1468:
    LexManos: Patch line number update. Ignore this.
    LexManos:
        New system in EntityRegistry to allow modders to register spawn eggs.
        For entites that do not use the global ID system.
        {Which no mod entity should}
        Vanilla spawn eggs will now detect a 'entity_name' entry in it's NBT data and use that for spawning/rendering.
```
- [X] Item registration
- [x] Item rendering
  - [x] Item model
  - [x] Item color
- [x] Item drops
- [x] Item crafting registration
- [x] That's what I forgot - spawner setting

**Comment by CAD97** [comment]
Should resolve #5 and #6 when migrating to the new system. Need to research how this affects #4 .

**Comment by CAD97** [comment]
Current progress fixes #6

**Comment by CAD97** [comment]
That finishes the task list. This issue is now closed as I notice no residue left over from before the switch.
